internal/website/docs/next/reference/hooks/usePreviewNode.mdx

```diff
@@ -0,0 +1,87 @@
+---
+slug: /next/reference/hooks/usePreviewNode
+title: usePreviewNode Hook
+description: The usePreviewNode hook provides a way to get post preview data from WordPress for any post type
+---
+
+The `usePreviewNode` hook provides an abstraction around getting preview data for any post type. When calling the `usePreviewNode` hook on an appropriate preview page, the hook will ensure authorization, make a request to determine the post type, then make a subsequent request to retrieve the proper preview data. When using 0.7.1 of FaustWP or later, preview data will be retrieved with one request.
```

Comment:
I believe the single query functionality would end up in FaustWP 0.7.2 or later since we're already on 0.7.1. That reminds me we need to add a changeset 🦋